programs/server/Server.cpp

```diff
@@ -513,7 +513,12 @@ if (ThreadFuzzer::instance().isEffective())
     // Initialize global thread pool. Do it before we fetch configs from zookeeper
     // nodes (`from_zk`), because ZooKeeper interface uses the pool. We will
     // ignore `max_thread_pool_size` in configs we fetch from ZK, but oh well.
-    GlobalThreadPool::initialize(config().getUInt("max_thread_pool_size", 10000));
+    GlobalThreadPool::initialize(
+        config().getUInt("max_thread_pool_size", 10000),
+        config().getUInt("max_thread_pool_free_size", 1000),
+        config().getUInt("thread_pool_queue_size", 10000),
+        config().getBool("thread_pool_shutdown_on_exception", false)
```

Comment:
I am convinced that this is bad idea (to allow the user to change this flag).

Reply:
I wanted to be consistent, but I’ll remove it. Thanks!